backend/src/services/response.ts

```diff
@@ -54,3 +63,81 @@ export async function updateResponse(user: UserInterface, responseId: string, co
 
   return response
 }
+
+export type ReviewResponseParams = Pick<ResponseInterface, 'comment' | 'decision'>
+export async function respondToReview(
+  user: UserInterface,
+  modelId: string,
+  role: string,
+  response: ReviewResponseParams,
+  kind: ReviewKindKeys,
+  reviewId: string,
+): Promise<ResponseInterface> {
+  const review = await findReviewForResponse(user, modelId, role, kind, reviewId)
+
+  // Store the response
```

Comment:
Is it worth checking if a review has already been approved? And perhaps throwing an error?